api/bazel/repository_locations.bzl

```diff
@@ -18,6 +18,10 @@ PROMETHEUS_SHA = "783bdaf8ee0464b35ec0c8704871e1e72afa0005c3f3587f65d9d6694bf391
 
 KAFKA_SOURCE_SHA = "ae7a1696c0a0302b43c5b21e515c37e6ecd365941f68a510a7e442eebddf39a1"  # 2.2.0-rc2
 
+# ZIPKINAPI_RELEASE = "0.2.2"
+ZIPKINAPI_GIT_SHA = "821bc013ecc730f4c76f33c50bd5e8f4b6fa8446"
```

Comment:
Will change this once it is tagged.

Reply:
cc. @adriancole @basvanbeek

Reply:
roger. as soon as you say go, we will merge and cut the patch